packages/itwinui-css/src/tabs/tabs.scss

```diff
@@ -191,3 +204,58 @@
     }
   }
 }
+
+@mixin iui-tabs-actions-wrapper {
+  grid-area: tabs-actions;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  // To always prevent actions from sticking to content
+  --_iui-borderless-horizontal-tabs-actions-content-gap: var(--iui-size-3xs);
+  --_iui-borderless-vertical-tabs-actions-content-gap: var(--iui-size-xs);
```

Comment:
these can be sass variables

Reply:
> `--_iui-borderless-horizontal-tabs-actions-content-gap` is being conditionally reassigned on [L220](https://github.com/iTwin/iTwinUI/pull/860/files#diff-0dba596e6381aa5a0bd620f0bb73c94ef6e65fcb3ce323f61bb25704a27a16adR220). So I thought of leaving that as a CSS variable.

but does it need to be reassigned? you can use different sass variables for borderless vs default

Reply:
I changed the css variables to scss variables (https://github.com/iTwin/iTwinUI/pull/860/commits/409c080d852ee797724d49c449f38e0b9c5e101d).

I had to change the selectors a little. Not sure which version is better. Also, this pattern is slightly different from the pattern followed in `expandable_block.scss` as mentioned in the https://github.com/iTwin/iTwinUI/pull/860#discussion_r1060950174 thread.

Also, not sure if these are the best variable names ([L218-220](https://github.com/iTwin/iTwinUI/pull/860/commits/409c080d852ee797724d49c449f38e0b9c5e101d#diff-0dba596e6381aa5a0bd620f0bb73c94ef6e65fcb3ce323f61bb25704a27a16adR218-R220)). Do you have any ideas for shorter names?

Reply:
> I had to change the selectors a little. Not sure which version is better. Also, this pattern is slightly different from the pattern followed in `expandable_block.scss` as mentioned in the [#860 (comment)](https://github.com/iTwin/iTwinUI/pull/860#discussion_r1060950174) thread.

ah yeah, so it's a tradeoff between shipping long css variables vs long selectors. i'm also not sure which one is better, but i'm fine with how it currently is.

> Also, not sure if these are the best variable names ([L218-220](https://github.com/iTwin/iTwinUI/pull/860/commits/409c080d852ee797724d49c449f38e0b9c5e101d#diff-0dba596e6381aa5a0bd620f0bb73c94ef6e65fcb3ce323f61bb25704a27a16adR218-R220)). Do you have any ideas for shorter names?

maybe `standard` should be `default` in the horizontal tabs vars. and for vertical it could be removed.
long sass variable names are ok to me, because they get compiled out in the final css.